src/transformers/models/depth_pro/__init__.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
`__init__` file should follow new standard, you can copy one from `align` model for example, + you will need to add `__all__ = [...]` to other files in the folder